src/v2/Apps/Artist/Components/ArtistHeader.tsx

```diff
@@ -80,191 +98,153 @@ export class ArtistHeader extends Component<Props> {
-                  </Sans>
-                )
-              }}
-              onOpenAuthModal={() =>
-                handleOpenAuth(props.mediator, props.artist)
-              }
-            >
-              Follow
-            </FollowArtistButton>
-          </Flex>
-        </Flex>
-        <Flex flexDirection="row" justifyContent="center">
-          {renderRepresentationStatus(props.artist)}
-          {renderAuctionHighlight(props.artist) &&
-            renderRepresentationStatus(props.artist) && <Spacer mr={0.5} />}
-          {renderAuctionHighlight(props.artist)}
-        </Flex>
-      </Flex>
-    )
-  }
-}
+            )}
 
-const handleOpenAuth = (mediator, artist) => {
-  openAuthToFollowSave(mediator, {
-    entity: artist,
-    contextModule: ContextModule.artistHeader,
-    intent: Intent.followArtist,
-  })
-}
+            {/**
+             * 0 followers is still displayed in this case:
+             * "displaying 0 helps the layout stay together"
+             */}
+            <Box mr={3}>
+              <Text as="dt" color="black60">
+                Followers
+              </Text>
 
-const renderAuctionHighlight = artist => {
-  const topAuctionResult = get(
-    artist,
-    () => artist.auctionResultsConnection.edges[0].node.price_realized.display
+              <Text as="dd">{formatFollowerCount(artist.counts.follows)}</Text>
+            </Box>
+          </Box>
+
+          <FollowArtistButton
+            useDeprecatedButtonStyle={false}
+            artist={artist}
+            user={user}
+            onOpenAuthModal={() => handleOpenAuth(mediator, artist)}
+          >
+            Follow
+          </FollowArtistButton>
+        </Box>
+      </Box>
+    </HorizontalPadding>
   )
-  if (topAuctionResult) {
-    const auctionLabel = topAuctionResult + " Auction Record"
-    return (
-      <ArtistIndicator
-        label={auctionLabel}
-        type="high-auction"
-        link={`/artist/${artist.slug}/auction-results`}
-      />
-    )
-  }
 }
 
-const renderRepresentationStatus = artist => {
-  const { artistHightlights } = artist
-  const { partnersConnection } = artistHightlights
-  if (
-    partnersConnection &&
-    partnersConnection.edges &&
-    partnersConnection.edges.length > 0
-  ) {
-    const highCategory = highestCategory(partnersConnection.edges)
+export const SmallArtistHeader: React.FC<ArtistHeaderProps> = ({
+  user,
+  artist,
+  mediator,
+}) => {
+  const topAuctionResult = getTopAuctionResult(artist)
+  const highCategory = getHighCategory(artist)
 
-    return (
-      <ArtistIndicator
-        label={CATEGORIES[highCategory]}
-        type={highCategory}
-        link={`/artist/${artist.slug}/cv`}
-      />
-    )
-  }
+  return (
+    <Box data-test={ContextModule.artistHeader} textAlign="center">
+      <Text as="h1" variant="largeTitle" mt={3} mb={1}>
+        {artist.name}
+      </Text>
+
+      {(artist.formattedNationalityAndBirthday ||
+        artist.counts.follows > 0) && (
```

Comment:
I didn't scrub through this whole but I have a general worry about these fields not existing/causing errors 🙈

Reply:
If you're talking about `counts` here — that can't actually be null since it's a nested object type, it always resolves: https://github.com/artsy/metaphysics/blob/e15eabafdf010d5d87a09e988c805aef38cdbe89/src/schema/v2/artist/index.ts#L476

Otherwise, I'm not accessing anything that wasn't already accessed here, there's appropriate guarding around.

Reply:
👍 cool!